modules/header/react/lib/parts/Search.tsx

```diff
@@ -255,13 +258,23 @@ export class Search extends React.Component<SearchProps, SearchState> {
       cursor: 'pointer',
     };
 
-    const {onSearchSubmit, ...props} = this.props;
+    const {
+      themeColor,
+      placeholder,
+      value,
+      rightAlign,
+      collapse,
+      onSearchSubmit,
+      ...elementProps
```

Comment:
Nit: not sure it's even worth calling out, but we've been using `elemProps` everywhere else.